notebooks/oracle2postgres/OracleToPostgres_notebook.ipynb

```diff
@@ -0,0 +1,962 @@
+    "    print(\"Bearer token generated\")\n",
+    "else:\n",
+    "    print(token)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5be3cf87-6d28-4b23-8466-87d3399f7a29",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "\n",
+    "oracle_to_pg_status = []\n",
+    "job_status_url = \"https://dataproc.googleapis.com/v1/projects/{}/locations/{}/batches/{}\"\n",
+    "for job in oracle_to_postgres_jobs:\n",
+    "    auth = \"Bearer \" + token[0]\n",
+    "    url = job_status_url.format(PROJECT,REGION,job)\n",
+    "    headers = {\n",
+    "      'Content-Type': 'application/json; charset=UTF-8',\n",
+    "      'Authorization': auth \n",
+    "    }\n",
+    "    response = requests.get(url, headers=headers)\n",
+    "    oracle_to_pg_status.append(response.json()['state'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1097575d-07c2-4659-a75f-d7e898e3f077",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "statusDF = pd.DataFrame({\"table\" : ORACLETABLE_LIST,\"oracle_to_postgres_job\" : oracle_to_postgres_jobs, \"oracle_to_postgres_status\" : oracle_to_pg_status})\n",
+    "statusDF"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0961f164-c7e4-4bb5-80f0-25fd1051147b",
+   "metadata": {},
+   "source": [
+    "## Step 11: Validate row counts of migrated tables from ORACLE to Postgres"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a3a28fb-3a39-4a10-b92d-0685b351a1b3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "oracle_row_count = []\n",
+    "postgres_row_count = []"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25299344-c167-4764-a5d1-56c1b384d104",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Get Oracle table counts\n",
+    "DB = sqlalchemy.create_engine(ORACLE_URL)\n",
+    "with DB.connect() as conn:\n",
+    "    for table in ORACLETABLE_LIST:\n",
+    "        results = DB.execute(\"select count(*) from {}\".format(table)).fetchall()\n",
+    "        for row in results:\n",
+    "            oracle_row_count.append(row[0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab0e539d-5180-4f5b-915e-35f7ea45e0d3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import psycopg2\n",
+    "postgresDB = psycopg2.connect(\n",
+    "                user=POSTGRES_USERNAME,\n",
+    "                password=POSTGRES_PASSWORD,\n",
+    "                dbname=POSTGRES_DATABASE,\n",
+    "                host=POSTGRES_HOST,\n",
+    "                port=POSTGRES_PORT\n",
+    "            )\n",
+    "\n",
+    "conn=postgresDB.cursor()\n",
+    "for table in ORACLETABLE_LIST:\n",
+    "    conn.execute('''select count(*) from {}'''.format(table))\n",
```

Comment:
You are missing `POSTGRES_SCHEMA` from this SQL statement. I get `UndefinedTable: relation "channels" does not exist` because I am not using `public for the schema`.
You need:
```
'select count(*) from {}.{}'.format(POSTGRES_SCHEMA, table)
```